lib/vsc/mympirun/mpi/openmpi.py

```diff
@@ -100,3 +101,55 @@ def pinning_override(self):
             self.log.raiseException('pinning_override: failed to write rankfile %s' % rankfn)
 
         return cmd
+
+    def make_machine_file(self):
```

Comment:
This is pretty much a copy-paste, only the contents of the file being written is different, right?

I'd clean this up by making `make_machine_file` in `mpi.py` accept a optional argument `nodes_txt`, so you can here do this:

```python
nodes_txt = ...
super(OpenMPI, self).make_machine_file(nodes_txt=nodes_txt)
```